src/SourceBuild/content/build.proj

```diff
@@ -19,18 +19,19 @@
   <UsingTask TaskName="Microsoft.DotNet.UnifiedBuild.Tasks.MergeAssetManifests" AssemblyFile="$(MicrosoftDotNetUnifiedBuildTasksAssembly)" TaskFactory="TaskHostFactory" />
   <Target Name="MergeAssetManifests" AfterTargets="Build">
     <PropertyGroup>
-      <MergedAssetManifestOutputPath>$(ArtifactsDir)VerticalManifest.xml</MergedAssetManifestOutputPath>
+      <MergedAssetManifestOutputPath>$(ArtifactsDir)$(VerticalName).xml</MergedAssetManifestOutputPath>
```

Comment:
When building locally, `VerticalName` is unset and this path will likely be invalid: `artifacts/.xml`.

Reply:
The unified build tests need this output path as well so I would move this property into the Directory.Build.props file.